smart-contract-verifier-http/tests/settings.rs

```diff
@@ -1,23 +1,28 @@
 use pretty_assertions::assert_eq;
 use smart_contract_verifier_http::Settings;
 
-// For linux systems we assume that all os specific values
-// are filled with defaults, so no need to rewrite them.
-#[cfg(target_os = "linux")]
-fn rewrite_os_specific_example_settings(
-    _example_settings: &mut Settings,
-    _default_settings: &Settings,
-) {
-}
-
-// For other systems we just use the values from default settings.
-#[cfg(not(target_os = "linux"))]
 fn rewrite_os_specific_example_settings(
     example_settings: &mut Settings,
     default_settings: &Settings,
 ) {
-    // For now, only server address is os system dependant
-    example_settings.server.addr = default_settings.server.addr;
+    example_settings.compilers.max_threads = default_settings.compilers.max_threads;
+    #[cfg(not(target_os = "linux"))]
+    {
+        use std::mem::discriminant;
+
+        example_settings.solidity.compilers_dir = default_settings.solidity.compilers_dir.clone();
```

Comment:
Nit: could you please add commented default `compilers_dir` for macos in `base.toml` and `base.env`, as it is currently done for `list_url`